tilavarauspalvelu/api/graphql/schema.py

```diff
@@ -216,9 +216,14 @@ class Query(graphene.ObjectType):
     )
     profile_data = Field(
         HelsinkiProfileDataNode,
-        reservation_id=graphene.Int(),
-        application_id=graphene.Int(),
-        description="Get information about the user, using Helsinki profile if necessary.",
+        reservation_id=graphene.Int(description="View profile data for this reservation user."),
+        application_id=graphene.Int(description="View profile data for this application user."),
+        description=(
+            "Get information about a user from Helsinki profile. "
+            "If user is not a profile user, still return known information. "
+            "Use only one of 'user_id', 'reservation_id' or 'application_id' to determine what user to get data for. "
+            "This determined required permissions to view the user's info."
```

Comment:
Typo?
-> `"This selection determines the permissions required to view the user's data."`